### PR TITLE
[WFCORE-3070] Found multiple secret keys sharing same CKA_LABEL

### PR DESCRIPTION
**Pull request by pskopek**
Synchronization issue when using FIPS enabled NSS DB as PKCS11 keystore.
https://issues.jboss.org/browse/WFCORE-3070
https://issues.jboss.org/browse/JBEAP-11693

### CODE REVIEW

File: server/src/main/java/org/jboss/as/server/mgmt/domain/HostControllerConnectionService.java [hunk 1]
```diff
@@ -206,23 +204,15 @@ private static SSLContext getAcceptingSSLContext() {
          * to be established.
          */
         try {
+            TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance("SunX509");
```
Comment (mchoma) [inline server/src/main/java/org/jboss/as/server/mgmt/domain/HostControllerConnectionService.java]: I wonder, will that work on IBM java??

**Comment by pskopek** [comment]
@mchoma I was getting exception that in FIPS mode only standard trust manager can be used.

**Comment by mchoma** [comment]
@pskopek , so probably you was hitting WFCORE-1135? Suficient solution to this was introduction of possibility to configure server fips compliant ssl context.  

Havent't had that custom trust manager implementation its place? I mean that empty check* methods? 

@darranl, can you have a look?

**Comment by pskopek** [comment]
@mchoma the problematic commit has been reverted

**Pull request merged**